lib/Conversation/CommentForm/CommentFormActions.js

```diff
@@ -30,7 +30,7 @@ const CommentFormActions = props =>
       </Button>
       <Button
         types={['secondary', 'size-sm']}
-        clickHandler={props.onSubmit}
+        clickHandler={() => {}}
         isSubmit
       >
         Send
```

Comment:
Why was the `props.onSubmit` removed in favour of an empty function here? I'm guessing it still makes sense to run the submit callback at this stage (and the cancellation one is still in place) :)